api/health/service.go

```diff
@@ -33,12 +38,24 @@ func (s *Service) Readiness(_ *http.Request, _ *struct{}, reply *APIReply) error
 }
 
 // Health returns a summation of the health of the node
-func (s *Service) Health(_ *http.Request, _ *struct{}, reply *APIReply) error {
+func (s *Service) Health(_ *http.Request, args *HealthArgs, reply *APIReply) error {
```

Comment:
I think we need to update the [client](https://github.com/ava-labs/avalanchego/blob/7d73b59cb4838d304387ea680b9cc4053b72620c/api/health/client.go#L22) to allow for this as well.

Reply:
updated